CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Fixed: bug causing rules in extended configs to be merged with, rather than replaced by, the extending config.
 - Fixed: bug where fractional keyframes yielded false positives in `selector-class-pattern`
 - Fixed: `selector-no-type` with secondary option `ignore: ["descendant"]` will now resolve nested selectors.
+- Fixed: `selector-max-specificity` now ignores selectors containing pseudo-classes with functional notation like `:unknown()`, and warns if the underlying `specificity` module cannot parse the selector. 
```

Comment:
- Fixed: `selector-max-specificity` now ignores selectors containing the `matches` pseudo-class, and warns if the underlying `specificity` module cannot parse the selector. 

I think this more accurately describes the current behaviour as only the `matches` pseudo-class is explicitly ignored in this PR.